pkg/scheduler/partition.go

```diff
@@ -339,26 +339,27 @@ func (pc *PartitionContext) AddApplication(app *objects.Application) error {
 		return fmt.Errorf("failed to find queue %s for application %s", queueName, appID)
 	}
 
-	// add the app to the queue to set the quota on the queue if needed
-	queue.AddApplication(app)
+	// set resources based on tags, but only if the queue is dynamic
+	if !queue.IsManaged() {
+		queue.SetResources(app.GetGuaranteedResource(), app.GetMaxResource())
```

Comment:
Should we keep the warnings "Trying to set max resources on a queue that is not an unmanaged leaf"?

Reply:
It's printed now